clang/test/Driver/sycl-offload-win.c

```diff
@@ -86,20 +85,18 @@
 
 /// ###########################################################################
 
-// RUN: touch %t.lib
-// RUN: %clang --target=x86_64-pc-windows-msvc -fsycl -foffload-static-lib=%t.lib %s -### 2>&1 \
-// RUN:   | FileCheck -DLIB=%t.lib %s -check-prefix=FOFFLOAD_STATIC_LIB_SRC2
-// RUN: %clang_cl --target=x86_64-pc-windows-msvc -fsycl -foffload-static-lib=%t.lib %s -### 2>&1 \
-// RUN:   | FileCheck -DLIB=%t.lib %s -check-prefix=FOFFLOAD_STATIC_LIB_SRC2
-// FOFFLOAD_STATIC_LIB_SRC2: clang-offload-bundler{{(.exe)?}}{{.+}} "-type=aoo" "-targets=sycl-spir64-{{.+}}-sycldevice" "-inputs=[[LIB]]"{{.+}} "-unbundle"
+// RUN: touch %t-orig.lib
```

Comment:
How exactly does such renaming help with Windows path issues?

Reply:
The problem we are dealing with is using `-DVAR=%t`.  As `%t` contains full directory information, this isn't translating well when being passed in.  The directory separators are not consistent causing miscompares, so I'm using additional strings in the file names to track within the output.